analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -454,7 +454,7 @@ private static void VerifyEmpty(string fileName, string snippet, DiagnosticAnaly
         {
             LanguageNames.CSharp => builder.WithLanguageVersion(Microsoft.CodeAnalysis.CSharp.LanguageVersion.Latest),
             LanguageNames.VisualBasic => builder.WithLanguageVersion(Microsoft.CodeAnalysis.VisualBasic.LanguageVersion.Latest),
-            _ => throw new UnexpectedLanguageException(language)
+            _ => throw new UnexpectedLanguageException(language.LanguageName)
```

Comment:
No.
Just no :)

Reply:
Accidental remain - see latest commit for actual change.